casepro/cases/tests.py

```diff
@@ -857,10 +858,25 @@ class PartnerCRUDLTest(BaseCasesTest):
     def test_read(self):
         url = reverse('cases.partner_read', args=[self.moh.pk])
 
-        # user from different partner but same org can see it
+        # manager user from same partner gets full view of their own partner org
+        self.login(self.user1)
+        response = self.url_get('unicef', url)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.context['can_manage'], True)
+        self.assertEqual(response.context['can_view_replies'], True)
```

Comment:
Might be worth constructing a more sophisticated way of managing permissions for partner users since these permissions differ when a partner user is accessing something tied to their own partner org.